src/ml_core/data_processor/time_series_data_processor.py

```diff
@@ -85,7 +90,12 @@ def _preprocessing(self):
         x_test, y_test = np.array(x_test), np.array(y_test)
 
         y_train = np.reshape(y_train, (y_train.shape[0], y_train.shape[1]))
-        y_test = np.reshape(y_test, (y_test.shape[0], y_test.shape[1]))
+
```

Comment:
prevent from training data ratio to be 1 and cause empty testing dataset stop the process